mathext/internal/cephes/hyp2f1.go

```diff
@@ -0,0 +1,408 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Cephes Math Library Release 2.8:  June, 2000
+Copyright 1984, 1987, 1992, 2000 by Stephen L. Moshier
+*/
+
+// Code derived from hyp2f1.c in https://www.moshier.net/cephes-math-28.tar.gz.
```

Comment:
Oh, sorry, I should have put a blank line below. This comment should be floating, not attached to the package decl.